claws/openclaw/sageox/test-skill.py

```
@@ -0,0 +1,203 @@
+]
+
+
+def send_prompt(prompt: str, session_id: str) -> dict:
+    """Send a prompt to OpenClaw via SSH and return the response."""
+    ssh_cmd = [
+        "ssh",
+        "-i", str(VPS_KEY),
+        "-o", "StrictHostKeyChecking=no",
+        "-o", "ConnectTimeout=10",
+        VPS_HOST,
+        f'openclaw agent --session-id {session_id} --message {repr(prompt)} 2>&1',
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)